benchmarks/dungeon-benchmark/src/main/java/com/squareup/sample/dungeon/benchmark/DungeonGatherBaselineProfile.kt

```diff
@@ -23,24 +23,21 @@ import kotlin.time.ExperimentalTime
  */
 @RunWith(AndroidJUnit4::class)
 @OptIn(ExperimentalBaselineProfilesApi::class)
-public class WorkflowBaselineProfiles {
+class DungeonGatherBaselineProfile {
```

Comment:
I don't understand what's up with dungeons and poetry. Can we use real names that explain the differences?